news/_posts/core-weekly/2019-07-31-coreweekly-week-29-2019.md

```diff
@@ -0,0 +1,129 @@
+* [#14587](https://github.com/PrestaShop/PrestaShop/pull/14587): Fix issue when searching for long words in FO, by [@matthieu-rolland](https://github.com/matthieu-rolland)
+
+
+### Tests
+
+* [#14231](https://github.com/PrestaShop/PrestaShop/pull/14231): PHPUnit no expectation annotation. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#14287](https://github.com/PrestaShop/PrestaShop/pull/14287): Behat tests for Category. Thank you [@sarjon](https://github.com/sarjon)
+* [#14593](https://github.com/PrestaShop/PrestaShop/pull/14593): First POM implementation, by [@SimonGrn](https://github.com/SimonGrn)
+* [#14612](https://github.com/PrestaShop/PrestaShop/pull/14612): Update docker for puppeteer, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#14725](https://github.com/PrestaShop/PrestaShop/pull/14725): Move travis scope from 5.6 to 7.1, by [@matks](https://github.com/matks)
+
+
+### Web services
+
+* [#8769](https://github.com/PrestaShop/PrestaShop/pull/8769): WS: Auto generate link_rewrite and do not require it when creating ca…. Thank you [@marekhanus](https://github.com/marekhanus)
+
+
+## Code changes in the "1.7.6.x" branch (for v1.7.6.0)
+
+
+### Back office
+
+* [#14673](https://github.com/PrestaShop/PrestaShop/pull/14673): Fix Orders view page link to transform guest to customer, by [@matks](https://github.com/matks)
+* [#14701](https://github.com/PrestaShop/PrestaShop/pull/14701): Fix log delete all action on Logs page, by [@matks](https://github.com/matks)
+
+
+## Code changes in modules, themes & tools
+
+### Shopping cart
+
+* [#41](https://github.com/PrestaShop/ps_shoppingcart/pull/41): The modal "add to cart confirmation" is not displayed on error. Thank you [@jf-viguier](https://github.com/jf-viguier)
+
+
+### Decimal
+
+* [#5](https://github.com/PrestaShop/decimal/pull/5): Add support for exponential notation, by [@eternoendless](https://github.com/eternoendless)
+
+
+### TranslationFiles
+
+* [#6](https://github.com/PrestaShop/TranslationFiles/pull/6): Add translations files for 1.7.7, update latest on 1.7.6, by [@jolelievre](https://github.com/jolelievre)
+
+
+### Customer reassurance block
+
+* [#20](https://github.com/PrestaShop/blockreassurance/pull/20): Major refactoring, by [@Progi1984](https://github.com/Progi1984)
+* [#21](https://github.com/PrestaShop/blockreassurance/pull/21): Some fixes for v4, by [@matks](https://github.com/matks)
+
+
+### classic-rocket
+
+* [#74](https://github.com/PrestaShop/classic-rocket/pull/74): Remove bs class and add utilities + make the button more ergonomic on phone. Thank you [@Kmoulun](https://github.com/Kmoulun)
+
+
+<hr />
+
+Thank you to the contributors whose pull requests were merged since the last Core Weekly Report: @202-ecommerce, @jf-viguier, @Kmoulun, @marekhanus, @MathiasReker, @rokaszygmantas, @sarjon, @SebBareyre, @webmak, @YeLnatSs, @zuk3975!
```

Comment:
Please add invertus in your script
```suggestion
Thank you to the contributors whose pull requests were merged since the last Core Weekly Report: @202-ecommerce, @jf-viguier, @Kmoulun, @marekhanus, @MathiasReker, @SebBareyre, @webmak, @YeLnatSs!
```

Reply:
we have always listed people who are not employees of the prestashop company.

Reply:
Nope, we don't add @khouloudbelguith for example.. https://github.com/PrestaShop/prestashop.github.io/blob/d943c89b0adb217d4d99d083dc4cbaf976517be5/news/_posts/core-weekly/2019-01-11-coreweekly-week-01-2019.md